### ISSUE TITLE
Throws ENOENT errors rather than failing gracefully when composed classname file is not found

### ISSUE THREAD

**Issue by ghost** [juanca/stylelint-css-modules#10]
When a `composes:` rule pointing to an external file is used and the file is not found, this plugin crashes Stylelint with an `ENOENT` error rather than failing gracefully.

For example, given a file `a.css`:
```css
.test {
  composes: whatever from "./non-existent-file.css";
```

Running `stylelint` yields:

`Error: ENOENT: no such file or directory, open '[path]/non-existent-file.css'`
(Plus a big stacktrace)

What's worse, when using an editor with Stylelint support, like VS Code with the [Stylelint plugin](https://marketplace.visualstudio.com/items?itemName=shinnn.stylelint), this causes errors on every single keypress:
![repor](https://user-images.githubusercontent.com/56890998/70260846-26e38b80-175f-11ea-8e29-c3b0b0224309.gif)

Expected behaviour: the plugin should report an error normally when a file is not found, rather than crashing stylelint entirely.

**Comment by juanca** [comment]
Good catch. Sorry for not catching this issue when it was reported. Looks like my notifications was not set up correctly. I'll try to get this fixed and released by EOW.

**Comment by juanca** [comment]
This will be released in the next published version, I just need to fix a few other issues.

**Comment by juanca** [comment]
See latest release @ https://github.com/juanca/stylelint-css-modules/releases/tag/1.1.0
